content/docs/hooks-overview.md

```diff
@@ -82,9 +82,9 @@ import React, { useState, useEffect } from 'react';
 function Example() {
   const [count, setCount] = useState(0);
 
-  // Similar to componentDidMount and componentDidUpdate:
+  // componentdidMount ve componentDidUpdate' e benzer bir şekilde:
```

Comment:
It would be better to change `componentDidUpdate' e` with `componentDidUpdate'e`.

Reply:
I have made the changes you requested